duckduckgo/BrowserServicesKit
# BSK SwiftLint fixes part 1: Autofix many SwiftLint warnings

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/0/1205234076299599/f
iOS PR: Drag and drop onto iOS project to test
macOS PR: Drag and drop onto macOS project to test
What kind of version bump will this require?: Patch

**Optional**:

Tech Design URL:
CC: @ayoy @bwaresiak 

**Description**:

This PR begins the process of fixing BSK's SwiftLint step on CI.

The repo currently has many hundreds of failures, so this is being done over a few PRs to make code review easier.



**Steps to test this PR**:
1. Check that BSK builds and runs on both iOS and macOS repo
1. Carefully inspect the diff and ensure that no actual changes are introduced - these PRs should only make cosmetic changes



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
